harness/testAtomics.js

```diff
@@ -44,6 +39,11 @@ function testWithAtomicsInBoundsIndices(f) {
     var good_indices = [
         (view) => 0/-1, // -0
         (view) => '-0',
+        (view) => undefined, // 0
+        (view) => Number.NaN, // 0
+        (view) => 3.5,
+        (view) => '3.5',
```

Comment:
Do you mean, if the length is 3 or less this will fail?  I'll change this to 0.5 which has the same effect for me, but may still fail if the length is zero.  I'll add a comment stipulating the minimum expected length.